tests/unit/models/blueprint-test.js

```diff
@@ -1528,6 +1528,27 @@ help in detail');
       expect(bluOpts).to.equal('-foo');
     });
 
+    it('sets blueprintOptions to saveDev by default', function() {
```

Comment:
Can we also add a test that checks the result of an actual `npm install` from a blueprint to see if the addon appears in `package.json`?